From 34122a921c07c881a195625d431617a140a2f59c Mon Sep 17 00:00:00 2001
From: Stefan Becker <chemobejk@gmail.com>
Date: Sun, 25 Oct 2020 14:53:46 +0200
Subject: [PATCH 10/50] programme: detect parental level with white space

It seems that some programme sources now use e.g. "( S )" instead of
"(S)". Add the optional white space to the regex.
---
 grab/fi/fi/programme.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grab/fi/fi/programme.pm b/grab/fi/fi/programme.pm
index 2d83c345..8a60c5e6 100644
--- a/grab/fi/fi/programme.pm
+++ b/grab/fi/fi/programme.pm
@@ -160,7 +160,7 @@ sub dump {
   # Programme post-processing
   #
   # Parental level removal (catch also the duplicates)
-  $title =~ s/(?:\s+\((?:S|T|K?7|K?9|K?12|K?16|K?18)\))+\s*$//
+  $title =~ s/(?:\s+\(\s*(?:S|T|K?7|K?9|K?12|K?16|K?18)\s*\))+\s*$//
       if $title_strip_parental;
   #
   # Title mapping
-- 
2.29.2

